covid19-etl/etl/idph_vaccine.py

```diff
@@ -18,6 +18,13 @@
 TOTAL_VACCINE_LINK = "https://idph.illinois.gov/DPHPublicInformation/api/covidvaccine/getStatewideVaccineDetails"
 
 
+def remove_time_from_date_time(str_datetime):
+    datetime_parts = str_datetime.split("T")
+    if len(datetime_parts) > 0:
```

Comment:
the result of `split` will always be of length > 0, so i think you can simplify the function to just `return str_datetime.split("T")[0]`

Reply:
just to be sure. The format can change anytime

Reply:
what i meant is that regardless of the values of `str1` and `str2`, `str1.split(str2)` will always contain at least 1 item afaik